src/main/java/uk/gov/hmcts/reform/civil/model/dq/Respondent1DQ.java

```diff
@@ -117,4 +118,9 @@ public StatementOfTruth getStatementOfTruth() {
     public FutureApplications getFutureApplications() {
         return respondent1DQFutureApplications;
     }
+
+    @JsonProperty("respondToClaimAdmitPartEmploymentTypeLRspec")
```

Comment:
If the Java field name and the json name is the same, neither JsonProperty or the getter should be needed

Reply:
this code was removed as removed complex type from ccd and backend